ryan-meinzer.txt

```diff
@@ -0,0 +1,91 @@
+https://codepen.io/ryanmeinzer/pen/rNarWbb
+
+    1. What is Semantic HTML? 
+    semantic = meaning
+    2. What is HTML used for? 
+    structuring language for the web and apps to grow
+    3. What is an attribute and where do we put it? 
+    it's a property that can change of a component, putting in html for example as a characteristic of a page element
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    h1 is the main hearder tag - can use on a page for the main heading but may not be practical to use it more than once
+    5. Name two tags that have required attributes
+    img and href
+    6. What do we put in the head of our HTML document? 
+    things that we want the browser to know/see but don't want the end user to see
+    7. What is an id?
+    a unique reference for an element (such as a section of a page) 
```

Comment:
Good answer, I like how you explicitly state 'unique'

Reply:
for sure, id's are one of the things I *think* I knew before Lambda